argocd-managed/prometheus/templates/secrets.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  name: kibana-credentials
+  namespace: prometheus-operator
```

Comment:
The namespace "prometheus" will be generated by the [Argocd application](https://github.com/bierteam/uber-kubernetes/pull/217/files#diff-f4263b8b8896d39064b966b2e3032af77e2e003c435082b527b297c0d4cb9cc8R45)
[It's the default in the template](https://github.com/bierteam/uber-kubernetes/blob/main/argocd-managed/argocd-apps/templates/application-template.yaml#L13)